blob/blob.go

```diff
@@ -308,9 +308,10 @@ func (b *Bucket) As(i interface{}) bool {
 	return b.b.As(i)
 }
 
-// ReadAll is a shortcut for creating a Reader via NewReader and reading the entire blob.
+// ReadAll is a shortcut for creating a Reader via NewReader with default
+// ReaderOptions, and reading the entire blob.
```

Comment:
Could also add `ReaderOptions` arg here and pass it through, WDYT?

Reply:
I don't feel like we should until we know what `ReaderOptions` will contain.